k8s/apply.sh.tpl

```diff
@@ -24,4 +24,4 @@ function guess_runfiles() {
 RUNFILES="${PYTHON_RUNFILES:-$(guess_runfiles)}"
 
 PYTHON_RUNFILES=${RUNFILES} %{resolve_script} | \
-  kubectl --cluster="%{cluster}" --context="%{context}" --user="%{user}" %{namespace_arg} apply -f -
\ No newline at end of file
+  kubectl --cluster="%{cluster}" --context="%{context}" --user="%{user}" %{namespace_arg} %{kubectl_args} apply -f -
```

Comment:
We should probably escape `kubectl_args` and all the other inputs. Here's how rules_go does it: https://github.com/bazelbuild/rules_go/blob/368049409d12d0827dcfd0bd401188c22651cc29/go/private/skylib/lib/shell.bzl

Example of loading it: https://github.com/bazelbuild/rules_go/blob/238329474a90d8ac8c128cec3184c515ef467c04/go/private/common.bzl#L18

Reply:
I concur on escaping. I'll look at these two examples.